docs/deployment/rds-s3/README.md

```diff
@@ -286,7 +286,81 @@ mysql> select * from observation_logs;
 
 4. Verify the bucket is not empty and was populated by outputs of the experiment.
 
-## 5.0 Uninstall Kubeflow
+## 5.0 (Optional) Setup RDS & S3 credentials access for notebooks & pipelines
+### Use Cases
+- Be able to use `boto3` or aws libraries that require to pass in credentials in a notebook and specify credentials without hard coding them. Access the credentials via environment variables.  
+- Be able to use [ml-metadata](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) in a notebook (eg: to explore metadata) and specify the credentials needed by using environment variables.
+- Be able to use [ml-metadata](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) to query metadata during a pipeline run by passing a kubernetes secret to a pipeline component.  
+- Be able to use `boto3` or aws libraries that require to pass in credentials in a Kubeflow Pipeline Component. 
+
+The following script takes care of setting up secrets for notebooks and pipelines with values from the secrets manager.  
```

Comment:
suggest some rewording to clarify things:

The following script creates a kubernetes `mysql-secret` and `mlpipeline-minio-artifact` secrets with RDS & S3 credentials specified in AWS Secrets Manager created while deploying the platform. This is a sample for demonstrating how you can use [PodDefault resource](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md) and secrets in Notebooks to access the metadata database and and artifacts in S3 bucket created by pipelines. Make sure you create separate database and IAM users and corresponding secrets in Secrets Manager for your users if you want fine grain access control and auditing.

Reply:
Thank you, done!